src/CMakeLists.txt

```diff
@@ -506,6 +520,12 @@ message("** Use OpenCV SIFT features: " ${OpenMVG_USE_OCVSIFT})
 
 message("\n")
 
+if (DEFINED OpenMVG_USE_INTERNAL_CEREAL)
+  message(STATUS "CEREAL: (internal)")
```

Comment:
While we are working on that, I wonder if we have a way to get the cereal version from the header file.
So when we are asking cmake log to our user we can diagnose more regarding the library version...